topics/dev/tutorials/community-tool-table/tutorial.md

```diff
@@ -0,0 +1,200 @@
+
+![A diagram illustrating the Galaxy Tool Metadata Extractor pipeline, showcasing the various steps involved in creating a community Galaxy tool table.](./images/galaxy_tool_metadata_extractor_pipeline.png "Workflow of the Galaxy Tool Metadata Extractor pipeline. Tool wrappers are parsed from different repositories and additional metadata is retrieved from bio.tools, BioConda, and the main public Galaxy servers. Upon filtering and manual curation of the data for specific scientific communities, the data is transformed into interactive web tables and a tool usage statistic-base word cloud, that can be integrated into any website.")
+
+The pipeline creates an [interactive table with all tools and their metadata](https://galaxyproject.github.io/galaxy_tool_metadata_extractor/). This table can be **filtered to only include tools that are relevant to a specific research community**. Here is an example for the microbial related tools:
+
+<iframe id="edam" src="https://galaxyproject.github.io/galaxy_tool_metadata_extractor/microgalaxy/" frameBorder="0" width="100%" height="600px"> ![Interactive table for microgalaxy tools](./images/microgalaxy_tools.png) </iframe>
+
+The generated community-specific interactive table can be used as it and/or embedded, e.g. into the respective Galaxy Hub page or Galaxy subdomain. This table allows further filtering and searching for fine-grained tool selection. 
+
+The pipeline is **fully automated** and executes on a **weekly** basis. Any research community can apply the pipeline to create a table specific to their community. 
+
+The aim is this tutorial is to create such table for a community.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Add your community Galaxy Tool Metadata Extractor pipeline
+
+To create a table for a community, you first need to create a folder in the `data/community` folder within [Galaxy Tool Metadata Extractor code source](https://github.com/galaxyproject/galaxy_tool_metadata_extractor). 
+
+> <hands-on-title>Create a folder for your community</hands-on-title>
+>
+> 1. Fork the repository [Galaxy Tool Metadata Extractor repository](https://github.com/galaxyproject/galaxy_tool_metadata_extractor) if not already done
+> 2. Go to the `data/communities` folder
+> 3. Click on **Add file** drop-down menu on the top
+> 4. Select **Create a new file**
+> 5. Fill in `Name of your file` with:  name of your community + `/categories`
```

Comment:
```suggestion
> 5. Fill in the `Name of your file` field with:  name of your community + `/categories`. This will create a new folder for your community and add a categories file to this folder.
```